devicon.json

```diff
@@ -1929,6 +1929,38 @@
         "color": "#59B3D8",
         "aliases": []
     },
+    {
+        "name": "couchbase",
+        "altnames": [],
+        "tags": [
+            "database",
+            "nosql",
+            "cloud"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#EA2328",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
```

Comment:
No need for an alias here anymore since `plain-wordmark.svg` exists, so that will be used instead of the original-wordmark alias
```suggestion
```